src/applications/simple-forms/mock-form-minimal-header/components/breadcrumbs.jsx

```diff
@@ -0,0 +1,35 @@
+import { VaBreadcrumbs } from '@department-of-veterans-affairs/component-library/dist/react-bindings';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const Breadcrumbs = (
+  <VaBreadcrumbs
+    className="breadcrumbs-container"
+    breadcrumbList={[
+      {
+        href: '/',
+        label: 'Home',
+      },
+      {
+        href: '/mock-form-minimal-header',
+        label: 'Mock form minimal header',
+      },
+    ]}
+    label="Breadcrumb"
+    homeVeteransAffairs={false}
+  />
+);
+
+export const CustomTopContent = ({ currentLocation }) => {
+  if (
+    currentLocation?.pathname.includes('confirmation') ||
+    currentLocation?.pathname.includes('introduction')
```

Comment:
Will this need to be configured/specified with every usage of minimal header?

Reply:
for now yes, maybe we can streamline this a little better when we have more examples in various forms.